src/lib/project_manager/project_manager.ts

```diff
@@ -836,14 +923,21 @@ class TreeDataProvider implements vscode.TreeDataProvider<TreeItem> {
     this.update_tree();
   }
 
-  get_vunit_test_list_items(vunit_test_list) {
-    let vunit_test_list_items: Test_item[] = [];
-    for (let i = 0; i < vunit_test_list.length; i++) {
-      const element = vunit_test_list[i];
+  get_test_list_items(test_list) {
+    let test_list_items: Test_item[] = [];
+    for (let i = 0; i < test_list.length; i++) {
+      const element = test_list[i];
       let item = new Test_item(element.name, element.location);
-      vunit_test_list_items.push(item);
+      if ("test_type" in element)
+      {
+        if (element.test_type !== undefined )
+        {
+          item.contextValue = `test_${element.test_type}`;
```

Comment:
@smgl9 here is a place where I expect to set `test_cocotb` or `test_vunit` to split action buttons. The error can be here.

Reply:
Found! https://github.com/TerosTechnology/vscode-terosHDL/pull/86/commits/0adc157a1e3245372b142379a9d60c5624581028